code/index.html

```diff
@@ -3,45 +3,93 @@
-        src="images/guess-who-bubble.png"
+        src="imagesnew/guessWhoLogoNew.png"
         alt="Guess Who"
       />
-
-      <h1>Does the person have</h1>
-
+      
+      <h1>Does the person have...</h1>
+      
       <select title="questions" id="questions">
+        <option value="default" disabled selected >choose an attribute</option>
         <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="brown">brown hair?</option>
+          <option value="blonde">blonde hair?</option>
+          <option value="no">no hair?</option>
+          <option value="black">black hair?</option>
+          <option value="grey">grey hair?</option>
+          <option value="white">white hair?</option>
+          <option value="red">red hair?</option>
+          <option value="very specific">very specific hair?</option>
+          
         </optgroup>
         <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+          <option value="brown">brown eyes?</option>
+          <option value="blue">blue eyes?</option>
+          <option value="haunting">haunting eyes?</option>
         </optgroup>
+
         <optgroup label="accessories">
-          <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="glasses">glasses?</option>
+          <option value="sunglasses">sunglasses?</option>
+          <option value="a headpiece">a headpiece?</option>
+          <option value="a dog">a dog?</option>
+          <option value="jewelry">jewelry?</option>
+          <option value="a weapon">a weapon?</option>
+          <option value="a plan">a plan?</option>
+          <option value="dollars">dollars?</option>
         </optgroup>
+        
         <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
+          <option value="a smoking habit">a smoking habit?</option>
+          <option value="a fashion habit">a fashion habit?</option>
+          <option value="a magical habit">a magical habit?</option>
+          <option value="a phone">braids?</option>
+          <option value="a beard">a beard?</option>
+          <option value="elf ears">elf ears?</option>
+          <option value="a mustache">a mustache?</option>
+          <option value="a striking complexion">a striking complexion?</option>
         </optgroup>
       </select>
 
       <button id="filter" class="filled-button">FIND OUT</button>
+
+      <section class="time-section">
+        <h2>Your time:
+          <span class="timer" id="timer">0</span> seconds
+        </h2>
+      </section>
+
+      <section class="count-section">
+        <h2>Number of guesses:
+          <span class="guess-container" id="guess-container">0</span>
+        </h2>
```

Comment:
Love that you have added a timer and number of guesses! It made me really feel the pressure 😅 As a way of making things more "complex" next time you could try and add these part through Javascript? Just if you want to try and evolve the app 🥰